pkg/util/errors/errors.go

```diff
@@ -5,5 +5,5 @@ var (
 	ErrInitServicesNotReady      = New(1001, "failed to initialize. Services are not ready")
 	ErrTimeoutServicesNotReady   = New(1002, "failed with timeout error.  Services are not ready")
 	ErrPeriodicCheck             = Newf(1003, "%s failed.  Services are not ready")
-	ErrStartingAgentStatusUpdate = New(1004, "error starting %s update")
+	ErrStartingAgentStatusUpdate = Newf(1004, "error starting %s update")
```

Comment:
good catch

Reply:
My errorcheck was failing. I mentioned it to Shane. I don't know why it wasn't caught when he put this message in.